modules/improvedigitalBidAdapter.js

```diff
@@ -23,33 +61,62 @@ export const spec = {
    * @param {object} bid The bid to validate.
    * @return boolean True if this is a valid bid, and false otherwise.
    */
-  isBidRequestValid: function (bid) {
+  isBidRequestValid(bid) {
     return !!(bid && bid.params && (bid.params.placementId || (bid.params.placementKey && bid.params.publisherId)));
   },
 
   /**
    * Make a server request from the list of BidRequests.
    *
    * @param {BidRequest[]} bidRequests A non-empty list of bid requests which should be sent to the Server.
+   * @param bidderRequest
    * @return ServerRequest Info describing the request to the server.
    */
-  buildRequests: function (bidRequests, bidderRequest) {
-    let normalizedBids = bidRequests.map((bidRequest) => {
-      return getNormalizedBidRequest(bidRequest);
-    });
-
-    let idClient = new ImproveDigitalAdServerJSClient('hb');
-    let requestParameters = {
-      singleRequestMode: (config.getConfig('improvedigital.singleRequest') === true),
```

Comment:
singleRequestMode must be implemented as in the old adapter, i.e. by default send a separate request per slot - I sent you a Slack comment about this